app/controllers/events/routes.py

```diff
@@ -13,6 +13,12 @@
 
 @events_bp.route('/events/<term>/', methods=['GET'])
 def events(term):
+    #set term to current term when events page is accessed from the navbar
```

Comment:
All of this could be replaced by `Term.get_or_none(isCurrentTerm = True)` or something like that. But... see overall comments.

Reply:
replaced line 18-21 with `Term.get_or_none(isCurrentTerm = True)`